src/library/scala/math/package.scala

```diff
@@ -219,6 +222,37 @@ package object math {
   /** @group signum */
   def signum(x: Double): Double = java.lang.Math.signum(x)
 
+
+  def floorDiv(x: Int, y: Int): Int = java.lang.Math.floorDiv(x, y)
+
+
+  def floorDiv(x: Long, y: Long): Long = java.lang.Math.floorDiv(x, y)
+
+  def floorMod(x: Int, y: Int): Int = java.lang.Math.floorMod(x, y)
+
+  def floorMod(x: Long, y: Long): Long = java.lang.Math.floorMod(x, y)
+
+  def copySign(magnitude: Double, sign: Double): Double = java.lang.Math.copySign(magnitude, sign)
+
+  def copySign(magnitude: Float, sign: Float): Float = java.lang.Math.copySign(magnitude, sign)
+
+  def nextAfter(start: Double, direction: Double): Double = java.lang.Math.nextAfter(start, direction)
+
+  def nextAfter(start: Float, direction: Float): Float = java.lang.Math.nextAfter(start, direction)
+
+  def nextUp(d: Double): Double = java.lang.Math.nextUp(d)
+
+  def nextUp(d: Float): Float = java.lang.Math.nextUp(d)
+
+  def nextDown(d: Double): Double = java.lang.Math.nextUp(d)
+
+  def nextDown(d: Float): Float = java.lang.Math.nextUp(d)
+
+  def scalb(d: Double, scaleFactor: Int): Double = java.lang.Math.scalb(d, scaleFactor)
+
+
+  def scalb(d: Float, scaleFactor: Int): Float = java.lang.Math.scalb(d, scaleFactor)
```

Comment:
Follow the Java convention of using `f` for float parameters here and in other methods.